state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -633,18 +641,32 @@ pub mod pallet {
 			Self::handle_broadcast_failure(broadcast_id, reporter.into())?;
 			Ok(().into())
 		}
+
+		/// Re-sign and optionally re-send a broadcast request.
+		///
+		/// Requires governance origin.
+		#[pallet::call_index(5)]
+		#[pallet::weight(Weight::zero())]
```

Comment:
I think it would be better to benchmark it properly or "over weight" it. Otherwise, this would open a D-Dos attack.

Reply:
This isn't the only place we do this... see PRO-770.